### DISCUSSION TITLE
Understanding WIDE, WBLK, 2020, and other recent additions

### DISCUSSION THREAD

**Discussion by MariusMasalar** [gopro/labs#266] [Q&A] [answered]
I recently upgraded my GP7 to the 11 and am trying to understand the latest Labs updates to tailor the camera to my needs. I'm running the current Labs firmware (2.10.70).

I'm very comfortable editing (mostly in Resolve), and I'm after the best quality footage I can get, mainly for use in good lighting outdoors, along with underwater in casual diving scenarios (<100ft, with good visibility). Not worried about recording long clips.

I've started with some basics:

- Camera set to high bitrate, Flat colour, with sharpening at low
- **BITR**=140 (Currently using a 256GB Sandisk Extreme V30—could I get away with higher values here safely?)
- **NR01**=80 (I'll play with this to see if I prefer to turn it down further and deal with noise entirely in post)

But now I'm wondering about some of the less familiar extensions and additions that I haven't found clear documentation for yet in [the usual spot](https://gopro.github.io/labs/control/extensions/):

- **WIDE**: Is this like a midpoint between the standard Flat profile and a custom curve applied via LOGB? Given my usage needs, would it make sense to apply this, something like LOGB=400, or stick with Flat?
- **WBLK**: Is this simply a 1 or 0 value that determines whether WB is locked when recording starts? If so it seems like something I'd want for general shooting too...have the camera adjust WB automatically until I hit record, then maintain that WB throughout the clip. Though I'm also wondering whether with 10-bit footage it makes sense to keep WB at Native and adjust in post instead?
- **2020 & MTRX**: Just generally curious about these two. What are the parameters and recommended usage?

Sorry for the pile of questions but I figured it's more efficient than making multiple posts. Thank you! 😊

**Comment by dnewman-gpsw** [comment, marked as answer]
BITR up to 170Mb/s is typically supported across all HERO11s with San Disk Extreme media.  Some can do 200Mb/s, but it can be scene dependent, so I generally don't exceed 180Mb/s.
NR01 all the way down to 1, if you do denoising in post. Although denoising 4 or 5K footage can require beefy GPUs.

WIDE - only touches the color gamut, not the transfer function (curve).  So you can shoot Natural + WIDE=1 if you want (although weird, it works.)   With WIDE off (0), the camera saturates color to the Rec709 gamut (basically sRGB), so some extreme real-world color saturations are clipped by the gamut. With WIDE=1, the gamut is sensor native, not Rec709, not Rec2020, just what the sensor sees.  This is like white balance set to Native - the difference is WIDE=1 leaves the white balancing active, making color correction a little easier and using more of the available bits (8/10-bit).
![image](https://user-images.githubusercontent.com/10856084/225715341-fe6e5a97-ed2c-42dd-9c2d-849d94561bbd.png)

WBLK - yes, you understand this function, it auto white balances until the capture starts.  The feature was added for scuba users, as fixed white balance are computed for above water, and while the auto works great in dive scenarios, some prefer a fix white balance per clip. Using Native WB is okay if you are shooting outdoors, it other lighting condition, WIDE=1 or a selected white balance is typically better.

2020 - do not use. Labs is filled with experiments, this was one of them that is not complete.  It will make a Rec2020 file (metadata), but clipped to Rec709 gamut, with Rec709 white point. So it doesn't do what you want.  

MTRX - super advanced.  The math that converts camera RGB into the YCrCb data that is encoded.  While crude attempts at Rec2020 might be possible (not recommended) you can use this for effects.  Want to shoot B&W, try oMMTRX=50,175,25,0,0,0,0,0,0  While doing B&W in post is easy, B&W in camera will use more of the bitrate to encode the image you want for a very subtle improvement. 
![image](https://user-images.githubusercontent.com/10856084/225721089-ac4b36d8-5a2b-4723-a066-6f75335bea53.png)

This is likely the best shooting mode for a wider gamut, a wider DR, from a GoPro, but only if you are prepared for a lot of post.
BITR=170, NR01=1, WIDE=1, LOGB=400 with ev at -2  As a QR     !MBITR=170!MNR01=1!MWIDE=1!MLOGB=400x-2

**Reply by andrerav** [reply]
Did a few quick tests with the above settings and this looks pretty freaking amazeballs!

**Reply by MariusMasalar** [reply]
Thank you for the quick and thorough answer, David! This is all super helpful and gives me a few different avenues to explore. Much appreciated. 😊

**Reply by gabisuk** [reply]
Hi guys, what's the 10 bit color management settings for this in davinci studio?

**Reply by tetrahydra** [reply]
Hi @dnewman-gpsw, that's a great explanation. We do know that the GoPro 11 sensor is tiny. Is it wise to use 170 Mbps bitrate as it only inflates information without gathering "real" lights? Higher bitrates = high space consumption, I could probably be wrong :-) I set my bitrate to 150Mbps just to balance out the need for "retaining useful" information and disk space.

I do edit and colour grade using FCP, but far from being a professional.

**Reply by dnewman-gpsw** [reply]
150Mb/s is good. Only extreme details with a lot of motion can use more.  FPV shooting can benefit from ~180Mb/s.
